velox/vector/BaseVector.cpp

```diff
@@ -734,7 +734,10 @@ uint64_t BaseVector::estimateFlatSize() const {
   }
 
   auto leaf = wrappedVector();
-  VELOX_DCHECK_GT(leaf->size(), 0);
+  // This can happen if wrapped vector has nulls in all rows?
+  if (UNLIKELY(leaf->size() == 0)) {
```

Comment:
Can we have a unit test for this? Thanks!

Reply:
@xiaoxmeng Of course, just tomorrow. Wanted to get this in for review, because I'm not sure this is the right fix.